### PR TITLE
Sorting user reviews #10

### LINKED ISSUE #10: User Story 17, User Show Page can sort reviews
```
As a Visitor,
When I visit a user's show page
I should also see links to sort reviews in the following ways:
- sort reviews newest first (descending chronological order)
- sort reviews oldest first (ascending chronological order)
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #12

child of #33

depends on #11

### PR DESCRIPTION
**Pull request by Vjp888**
Implements the review sort function for the user showpage.

The show page now has two links that will allow a user to sort the page based on the review's created_at time stamp. 

all tests passing.

closes #10

### CODE REVIEW

File: app/controllers/reviews_controller.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class ReviewsController < ApplicationController
 
   def show
+    # binding.pry
```
Comment (milevy1) [inline app/controllers/reviews_controller.rb]: Can you delete this binding.pry?

**Comment by milevy1** [comment]
Looks good. Remove that commented binding.pry and then I approve to merge.
